src/components/molecules/MoleculeSearchBox.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="search-control">
+        <p></p>
+        <atom-input v-model="searchModel" :placeholder="placeholder">
```

Comment:
use camelCase for the component name ... refer for the good practice https://medium.com/@riccardopolacci/vue-js-best-practices-c5da8d7af48d